packages/aws-cdk-lib/scripts/airlift-custom-resource-handlers.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+scriptdir=$(cd $(dirname $0) && pwd)
+customresourcedir=${scriptdir}/../../custom-resource-handlers
+awscdklibdir=${scriptdir}/..
+
+list_custom_resources() {
+  for dir in $(ls $customresourcedir/lib); do
+    [ -d $customresourcedir/lib/$dir ] && echo lib/$dir/$(ls $customresourcedir/lib/$dir)
+  done
+}
```

Comment:
This thing is weird. What is this doing?

First of all, instead of

```
for dir in $(ls $customresourcedir/lib);
```

You could  do:

```
  for dir in $customresourcedir/lib/*;
```

Second of all, it seems to be assuming there's only ever one file inside each of the directories. If there were 2, you would get:

```
echo lib/$dir/index.js helper.js
```

Which is probably not what you intended.

So why not assume that all entrypoints must always be called `index.ts` and go:


```
list_custom_resources() {
  echo lib/*/*/index.ts
}
```

?

Reply:
a big part of it is just getting the file path after `lib` because that is how i am going to store it in the generated folder as well. but with a few modifications, this works